recipe/meta.yaml

```diff
@@ -1,18 +1,17 @@
 {% set name = "openldap" %}
-{% set version = "2.6.4" %}
+{% set version = "2.6.10" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: https://www.openldap.org/software/download/OpenLDAP/openldap-release/{{ name }}-{{ version }}.tgz
-  sha256: d51704e50178430c06cf3d8aa174da66badf559747a47d920bb54b2d4aa40991
+  url: https://www.openldap.org/software/download/OpenLDAP/{{ name }}-release/{{ name }}-{{ version }}.tgz
+  sha256: c065f04aad42737aebd60b2fe4939704ac844266bc0aeaa1609f0cad987be516
 
 build:
   number: 0
   skip: True  # [not unix]
-  skip: True  # [linux and s390x]
   run_exports:
     - {{ pin_subpackage('openldap', max_pin='x.x') }}
 
```

Comment:
What is the error if you add  `{{ stdlib("c") }}` to `build`?

Reply:
The logs show the failing command had no `-I$PREFIX/include`, hence `openssl/opensslv.h` not found. Adding the include path to `CPPFLAGS` fixes that.

I tried to build with `{{ stdlib("c") }}` and with `CPPFLAGS` on `osx-arm64` locally and it succeeded:
```
if [[ "$(uname)" == "Darwin" ]]; then
  export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include"
fi
```